packages/axes/src/components/Axes.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { Axis } from './Axis'
+import { AnyScale, AxisProp } from '../types'
+
+const positions = ['top', 'right', 'bottom', 'left'] as const
+
+export const Axes = <X extends number | string | Date, Y extends number | string | Date>({
```

Comment:
I think we could create an alias for `number | string | Date`, it would be easier to maintain.